bicycleparameters/main.py

```diff
@@ -35,8 +35,8 @@ def __new__(cls, bicycleName, pathToData='.', forceRawCalc=False,
         try:
             pathToBicycle = os.path.join(pathToData, 'bicycles', bicycleName)
             if os.path.isdir(pathToBicycle) == True:
-                print("We have foundeth a directory named: " +
-                      "{0}.".format(pathToBicycle))
+                '''print("We have foundeth a directory named: " +
+                      "{0}.".format(pathToBicycle))'''
```

Comment:
Undo this comment. We can remove these notifications but they should be addressed in a separate pull request.